pkg/crc/preflight/preflight.go

```diff
@@ -163,6 +163,11 @@ func StartPreflightChecks(config config.Storage) error {
 func SetupHost(config config.Storage) error {
 	experimentalFeatures := config.Get(cmdConfig.ExperimentalFeatures).AsBool()
 	mode := network.ParseMode(config.Get(cmdConfig.NetworkMode).AsString())
+	if mode == network.VSockMode {
```

Comment:
I don't think it should be here. Preflight checks configure the host so that the daemon can run. 
If vsock is turned off, daemon will not start. And with this if, it will be impossible to enable it with crc setup.
Perhaps just put it near crc start?

Reply:
@guillaumerose Does `crc daemon` also need `crc setup` before ?

Reply:
Yes